common/views/components/SearchFilters/SearchFiltersDesktop.tsx

```diff
@@ -208,11 +208,9 @@ const DynamicFilterArray = ({
     }
   };
   useEffect(() => {
-    if (isNewStyle) {
```

Comment:
This component is already only displayed under `isNewSearch` condition, so no need to check here